pkg/deploy/ingress.go

```diff
@@ -27,11 +27,8 @@ func NewIngress(cr *orgv1.CheCluster, name string, serviceName string, port int)
 	}
 	ingressDomain := cr.Spec.K8SOnly.IngressDomain
 	ingressClass := util.GetValue(cr.Spec.K8SOnly.IngressClass, DefaultIngressClass)
-	labels := GetLabels(cr, util.GetValue(cr.Spec.Server.CheFlavor, DefaultCheFlavor))
+	labels := GetLabels(cr, name)
```

Comment:
is name never equal to rh-sso or identityprovider? always "keycloak" ?

Reply:
no, it's always called `keycloak` according to the code.

Reply:
hmm. we did a big refactoring in CRW to replace keycloak with identityProvider so I was concerned we'd done and incomplete job over in the che-operator. if you're satisfied, so am I. (It's not ideal to always refer to the identityProvider as keycloak when it can be something else... but neither is it ideal to call the postgres image "postgres" when it can in theory be replaced with another DB - https://issues.jboss.org/browse/CRW-224 for example.

Reply:
anyway refactoring can be a future fix for Che 8. :)

Reply:
OK. Thanks.